data/portfolioinfo2.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "companyName": "Hilton",
+    "companyLink": "http://stayhgv.com/",
+    "experience": "<p>Candy canes donut lemon drops gummies caramels. Candy canes cake cupcake toffee jelly beans jelly beans powder. Pastry dragée sweet roll jujubes pastry muffin lemon drops toffee. Cotton candy candy oat cake dragée candy canes jelly.</p>"
+  },
+  {
+    "companyName": "Marriott",
+    "companyLink": "http://www.marriottvacationclub.com/",
+    "experience": "<p>Sweet soufflé bonbon pie chocolate. Liquorice chupa chups cake chupa chups donut. Candy sesame snaps carrot cake carrot cake fruitcake jujubes powder. Tart jelly beans chocolate bar tart pie jelly-o jelly beans.</p>"
+  },
+  {
+    "companyName": "KDDI Evolva",
+    "companyLink": "http://www.k-evolva.com/eng/",
+    "experience": "<p>Ice cream danish toffee sesame snaps chocolate gingerbread oat cake sesame snaps. Fruitcake sesame snaps pudding gummi bears soufflé. Sweet roll jujubes cake lollipop croissant pudding. Croissant jujubes dragée marzipan brownie.</p>"
+  },
```

Comment:
Nope, I changed it to show not an actual data.

Thank you,
Sawako
> On Jan 25, 2016, at 10:04 AM, Mary Dickson <notifications@github.com> wrote:
> 
> In data/portfolioinfo2.json <https://github.com/ishidas/jan2016-portfolio/pull/10#discussion_r50730656>:
> 
> > +[
> > +  {
> > +    "companyName": "Hilton",
> > +    "companyLink": "http://stayhgv.com/",
> > +    "experience": "<p>Candy canes donut lemon drops gummies caramels. Candy canes cake cupcake toffee jelly beans jelly beans powder. Pastry dragée sweet roll jujubes pastry muffin lemon drops toffee. Cotton candy candy oat cake dragée candy canes jelly.</p>"
> > +  },
> > +  {
> > +    "companyName": "Marriott",
> > +    "companyLink": "http://www.marriottvacationclub.com/",
> > +    "experience": "<p>Sweet soufflé bonbon pie chocolate. Liquorice chupa chups cake chupa chups donut. Candy sesame snaps carrot cake carrot cake fruitcake jujubes powder. Tart jelly beans chocolate bar tart pie jelly-o jelly beans.</p>"
> > +  },
> > +  {
> > +    "companyName": "KDDI Evolva",
> > +    "companyLink": "http://www.k-evolva.com/eng/",
> > +    "experience": "<p>Ice cream danish toffee sesame snaps chocolate gingerbread oat cake sesame snaps. Fruitcake sesame snaps pudding gummi bears soufflé. Sweet roll jujubes cake lollipop croissant pudding. Croissant jujubes dragée marzipan brownie.</p>"
> > +  },
> is this working? make sure to remove the last comma in an array for json files or it can break everything
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ishidas/jan2016-portfolio/pull/10/files#r50730656>.
>